patches/chrome-browser-ui-webui-settings-about_handler.cc.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/browser/ui/webui/settings/about_handler.cc b/chrome/browser/ui/webui/settings/about_handler.cc
+index c372368a1e0abe7d9ee2af13208f328ea4416c1f..240de00ea312f01ea82028e8af3ed83bf82a22b3 100644
+--- a/chrome/browser/ui/webui/settings/about_handler.cc
++++ b/chrome/browser/ui/webui/settings/about_handler.cc
+@@ -282,6 +282,16 @@ AboutHandler::~AboutHandler() {
+ 
+ AboutHandler* AboutHandler::Create(content::WebUIDataSource* html_source,
+                                    Profile* profile) {
++  std::string brave_product_version = version_info::GetVersionNumber();
```

Comment:
I believe this logic already exists somewhere and if it doesn't it needs to be put in a helper. It definitely doesn't belong here in this patch